lib/util/merge-confidence/index.ts

```diff
@@ -37,32 +37,23 @@ export function initConfig(): void {
 }
 
 export function parseSupportedDatasourceString(): string[] | undefined {
-  const supportedDatasourceString =
-    process.env.RENOVATE_X_MERGE_CONFIDENCE_SUPPORTED_DATASOURCES;
+  const supportedDatasources = GlobalConfig.get(
+    'mergeConfidenceSupportedDatasources',
+  );
 
-  if (!is.string(supportedDatasourceString)) {
+  if (is.nullOrUndefined(supportedDatasources)) {
     return undefined;
   }
 
-  let parsedDatasourceList: unknown;
-  try {
-    parsedDatasourceList = parseJson(supportedDatasourceString, '.json5');
-  } catch (err) {
-    logger.error(
-      { supportedDatasourceString, err },
-      'Failed to parse supported datasources list; Invalid JSON format',
-    );
-  }
-
-  if (!is.array<string>(parsedDatasourceList, is.string)) {
+  if (!is.array<string>(supportedDatasources, is.string)) {
```

Comment:
```suggestion
  if (!is.array(supportedDatasources, is.string)) {
```
I think we don't need this type info?